docs/tutorial/code/go/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func helloWorldHandler(w http.ResponseWriter, req *http.Request) {
+	log.Printf("new hello world request")
+	fmt.Fprintf(w, "Hello, world!\n")
```

Comment:
Nit: slight simplification:
```suggestion
	fmt.Fprintln(w, "Hello, world!")
```